purchase_requisition_notes/tests/TEST_INSTRUCTIONS.rst

```diff
@@ -0,0 +1,5 @@
+- Open Purchase > Orders > Blanket Orders
+- Create a new Blanket Order
+- Under the Notes tab enter a note
+- Create New Quotation, and check the Notes tab, the note from Blanket order is copied
+- Confirm the Purchase Order, open the Notes tab, the note from Blanket order is copied
```

Comment:
The last check does not make sense. It should be copied only once the requisition is set or selected.